runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -994,4 +1006,13 @@ public void testMultikill()
 		assertEquals("Suqahs", slayerPlugin.getTaskName());
 		assertEquals(229, slayerPlugin.getAmount()); // 2 kills
 	}
+
+	// Helper method for ConfigManager mock
+	// Last call to the manager needs to have the expected value
+	private void verifyIntConfig(String key, Integer expectedValue)
+	{
+		ArgumentCaptor<Integer> captor = ArgumentCaptor.forClass(int.class);
+		verify(configManager, atLeastOnce()).setRSProfileConfiguration(eq(SlayerConfig.GROUP_NAME), eq(key), captor.capture());
```

Comment:
Why was this changed to `atLeastOnce()`?

Reply:
In case the config gets set more than once, the only thing that matters is that the last call is of the expected value. Without `atLeastOnce()`, the captor serves no purpose and could just be a full equality check as the default behavior is that the call needs to happen exactly once.